.travis.yml

```diff
@@ -11,6 +11,7 @@ before_install:
   - gem --version
   - bundle -v
 script:
+  - 'bundle exec puppet --version'
```

Comment:
oh snap I forgot about that from bundler.  I'll just remove it without updating module-sync.